src/Microsoft.Crank.RegressionBot/Program.cs

```diff
@@ -359,19 +366,16 @@ private static async Task CreateRegressionIssue(IEnumerable<Regression> regressi
 
             var title = await CreateIssueTitle(regressions, titleTemplate);
 
-            if (!_options.Debug)
```

Comment:
Is this not helpful to have?

Reply:
No, it's hiding too much, could repro a bug with it, and without it would do too few.